department-of-veterans-affairs/va.gov-cms
# VACMS-8291 kb login

## Description
Closes #8291. Closes #8289. 

- We got overzealous on the new login page, Dave requested the one link with updated verbiage to direct users into the KB first before contacting help desk.
- Re-placed & re-exported the login block... appeared to be suffering from the same UUID issue that the cms announcement block did.
- Fixed styles for the login form that appears to anonymous users on KB pages
- Fixed the placeholder color for the search form in the header on KB pages. no ticket, this was pointed out by Dave during the CMSx backlog refinement today.

## Testing done
as a logged out user:
- verified there was only one link on the main log in page.
- Verified KB page had login available after following the link.
- Verified toggle works on KB login. 
- Verified the placeholder in the header KB search form passes contrast

## Screenshots
<img width="314" alt="Screen Shot 2022-03-08 at 4 12 59 PM" src="https://user-images.githubusercontent.com/11279744/157348780-dca3b838-75e8-4930-9f72-860e7efc195f.png">
<img width="294" alt="Screen Shot 2022-03-08 at 4 13 05 PM" src="https://user-images.githubusercontent.com/11279744/157348785-437f4262-69a1-44cf-8739-e7826099f80d.png">
<img width="524" alt="Screen Shot 2022-03-08 at 4 13 16 PM" src="https://user-images.githubusercontent.com/11279744/157348787-46efdc84-0000-4ac8-9845-7f0fdeebfe60.png">


## QA steps

- visit the login page, 
-  [x] see that the "get help" link is present on both PIV and user/pass login flows.
- [x] Click the "get help" link, see that the login form is present in the sidebar on KB pages
- [x] See that the search bar in the KB header has a placeholder that passes a11y contrast.

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
